tests/mocha/standalone.js

```diff
@@ -0,0 +1,19 @@
+import assert from 'assert'
+import { remote } from '../../packages/webdriverio'
+import { runSync } from '../../packages/wdio-sync'
+
+describe('scripts run in standalone mode', () => {
+    it('should allow to run standalone mode synchronously', async () => {
+        return remote({
```

Comment:
Would this work too?
```
const client = await remote({
  runner: true,
  hostname: 'localhost',
  port: 4444,
  path: '/',
  capabilities: {
    browserName: 'chrome'
  }
})

runSync(() => {
  assert.equal(client.getTitle(), 'Mock Page Title')
})
```

Reply:
no, runSync requires an onReject and onResolve method, this would work:

```js
const client = await remote({
  runner: true,
  hostname: 'localhost',
  port: 4444,
  path: '/',
  capabilities: {
    browserName: 'chrome'
  }
})

await new Promise((resolve, reject) => runSync(() => {
  assert.equal(client.getTitle(), 'Mock Page Title')
})(resolve, reject)
```